lib/ts/recipe/recipeModule/types.ts

```diff
@@ -68,3 +68,9 @@ export type RecipeFunctionOptions = {
         userContext: any;
     }) => Promise<{ url: string; requestInit: RequestInit }>;
 };
+
+export type UserType = {
+    id: string;
+    email: string;
+    timeJoined: number;
+};
```

Comment:
Don't have this here. Have it inside the specific recipe only

Reply:
Moved it to emailpassword types